src/_P004_Dallas.ino

```diff
@@ -612,16 +616,16 @@ uint8_t Plugin_004_DS_read_bit(void)
     #if defined(ESP32)
        ESP32noInterrupts();
     #endif
-    pinMode(Plugin_004_DallasPin, OUTPUT);
     digitalWrite(Plugin_004_DallasPin, LOW);
-    delayMicroseconds(3);
+    pinMode(Plugin_004_DallasPin, OUTPUT);
```

Comment:
The same here, why set to output after writing to it?

Reply:
Because i wanna have a known output state set before activating the drivers.
Maybe i'm wrong with the esp8266 but i found it's good practice with other processors.